### ISSUE TITLE
I think I have a defective build of Psych Engine (with my source code modifications)

### ISSUE THREAD

**Issue by PDL2CGamejolt** [ShadowMario/FNF-PsychEngine#11638]
### Describe your bug here.

(Central by Null_y34r) The game crashes at around 2 minutes
Video: https://drive.google.com/file/d/1paYHx3io1Euqvzg1HU6DeS0y54-9iV5J/view?usp=sharing

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

I used both Source and Lua to make the mod

**Comment by 40U538072** [comment]
Of course, it is defective. Why not? Psych Engine is a crappy engine with bugs the devs will not fix. It is best if you give up. Don't mean to be harsh. Just a suggestion.

**Comment by PDL2CGamejolt** [comment]
"Give up"? As in- SWITCH?

**Comment by PDL2CGamejolt** [comment]
Also the build I modified is the 0.6.2 build.

**Comment by PDL2CGamejolt** [comment]
I'm switching to OS Engine.

**Comment by Yuta12342** [comment]
> Of course, it is defective. Why not? Psych Engine is a crappy engine with bugs the devs will not fix. It is best if you give up. Don't mean to be harsh. Just a suggestion.

What is your problem with Psych Engine, really?

**Comment by TheKitBoi** [comment]
> I'm switching to OS Engine.

can’t wait till he sends issue about why his software fucked and didn’t say about os engine

**Comment by TheKitBoi** [comment]
> > Of course, it is defective. Why not? Psych Engine is a crappy engine with bugs the devs will not fix. It is best if you give up. Don't mean to be harsh. Just a suggestion.
> 
> What is your problem with Psych Engine, really?

it’s started from this https://github.com/ShadowMario/FNF-PsychEngine/issues/11630

**Comment by PDL2CGamejolt** [comment]
The thing keeps crashing. (I switched to OS for that reason)

**Comment by 40U538072** [comment]
> > > Of course, it is defective. Why not? Psych Engine is a crappy engine with bugs the devs will not fix. It is best if you give up. Don't mean to be harsh. Just a suggestion.
> > 
> > 
> > What is your problem with Psych Engine, really?
> 
> it’s started from this #11630

Nope. It started with #11334
And I honestly can't stand how rude and how much of a jerk you are ever since I saw you. What is your problem with all of this? Why can't people like you work with me here and take things seriously, instead of just treating somebody like I am the retarded bonehead?

**Comment by CrusherNotDrip** [comment]
> And I honestly can't stand how rude and how much of a jerk you are ever since I saw you. 

how is he a jerk if you just called an engine crappy and just assume devs dont fix anything
you also still decided to go on the *crappy* engine github anyways

**Comment by 40U538072** [comment]
> > And I honestly can't stand how rude and how much of a jerk you are ever since I saw you.
> 
> how is he a jerk if you just called an engine crappy and just assume devs dont fix anything you also still decided to go on the _crappy_ engine GitHub anyways even after you said you hated the engine in another issue

It was on November 29, 2022. One of my issues I opened, I received a comment from TheKitBoi saying "low pc issue". He later deleted the comment, but I have history of it on my email. 9 times out of 10, he was literally being a jerk. The only issue he really tried to help me with was #11364, which his ideas didn't fix my audio issues, but I gave them a try. After that, I didn't receive any more help or any more replies back and on the 1st of December, I had to close the issue since it wouldn't fix it. I still have that issue and haven't been able to fix it ever since finding it.
